test/run-tests.proj

```diff
@@ -10,7 +10,7 @@
     <Message Importance="High" Text="Running tests ..." />
     <MSBuild Targets="VSTest"
              Projects="@(Project)"
-             Properties="VSTestNoBuild=true" />
+             Properties="VSTestNoBuild=true;Coverage=true;" />
```

Comment:
There should be a way to shut this off locally, if devs don't want it.

Reply:
Correct, that option needs to be added to the run tool and removed here.